dsl/camel-jbang/README.md

```diff
@@ -9,7 +9,7 @@ jbang app install camel@apache/camel
 If you however like to install camel-jbang from this project build you create an alias to the local entry point.
 
 ```shell
-jbang alias add --name camel -Dcamel.jbang.version=4.8.3-SNAPSHOT ./dsl/camel-jbang/camel-jbang-main/dist/CamelJBang.java
+jbang alias add --name camel -Dcamel.jbang.version=4.8.3-SNAPSHOT $HOME/git/camel/dsl/camel-jbang/camel-jbang-main/dist/CamelJBang.java
```

Comment:
Thomas this is maybe something that is your computer only with having a git folder in home. I think the readme should be neutral